barman/cli.py

```diff
@@ -890,6 +890,18 @@ def rebuild_xlogdb(args):
             help="The name of the AWS region containing the EC2 VM and storage "
             "volumes for recovery of a snapshot backup",
         ),
+        argument(
+            # TODO: change name of flag after definition
+            "--local-staging-path",
+            dest="local_staging_path",
```

Comment:
I thought about another implementation that would reuse the `recovery_staging_path` here in the `dest=recovery_staging_path` instead of using the new field. We could make --local-stating-path required for the recover of incremental backups. Then if there is a recovery_staging_path set in the configuration files, we would ignore it for incremental backups and warn the user that it needs to set `--local-staging-path`. By doing this, it would be hidden from the user that we are actually reusing the recovery_staging_path and we would have a new flag.
This way we would not need to create a new configuration field and it would be something like aliasing the existing configuration field. 

Let me know what you guys think, please! @barthisrael @gcalacoci @martinmarques @gustabowill  @bleidens

Reply:
we already discussed and the best approach would be to have another configuration field, instead of reusing it

Reply:
We are going with `--local-staging-path` after discussing this in the checkpoint call we had today.